fl4health/parameter_exchange/fedpm_exchanger.py

```diff
@@ -0,0 +1,20 @@
+from typing import Optional
+
+import torch
+import torch.nn as nn
+from flwr.common.typing import Config, NDArrays
+
+from fl4health.parameter_exchange.layer_exchanger import DynamicLayerExchanger
+from fl4health.utils.functions import sigmoid_inverse
+
+
+class FedPmExchanger(DynamicLayerExchanger):
```

Comment:
Thanks for bringing up the alternative! After thinking about it I decided to go with the selection function approach because I think it extends the existing functionality quite nicely. However, I rewrote the `__init__` method for `FedPMExchanger` since it is designed to be used specifically with `select_scores_and_sample_masks`, while originally the user needs to pass in the selection function themselves. I think now it makes more sense and hopefully reduces some confusion 🙂

Reply:
Sounds good!